roles/openstackclient/templates/openstack.j2

```diff
@@ -2,17 +2,8 @@
 
 # DO NOT EDIT THIS FILE BY HAND -- YOUR CHANGES WILL BE OVERWRITTEN
 
-INTERACTIVE=${INTERACTIVE:-false}
 PARAMETERS=${PARAMETERS:-}
 
-if [[ -t 0 ]]; then
```

Comment:
This tests whether stdin is a terminal. So if you want to have the output in unix format, just invoke `openstackclient </dev/null`

Reply:
We can either leave the current settings and remove the \r\n from output:
`for i in $(openstack server list -c ID -f value | tr -d "\r"); do openstack server show $i; done`

Or we change the default behaviour of the docker-wrapper-scripts to not containing carriage-returns for easier scripting usability (and maybe add an additional flag if you really want to run an interactive session?)

Reply:
Not sure what is wrong with adding the `</dev/null`, but maybe https://github.com/osism/ansible-collection-services/pull/1268 suits you better?